### ISSUE TITLE
Slow start

### ISSUE THREAD

**Issue by michalpaukert** [gitextensions/gitextensions#6441]
## Current behaviour

In current state I have just 3 small repositories and it takes almost 15 seconds before UI is displayed.

## Expected behaviour
I would expect to display UI much faster

## Steps to reproduce

Start git extensions

## Screenshots
![image](https://user-images.githubusercontent.com/36043389/55624141-56439200-57a5-11e9-879f-fc8c09a68263.png)

## Did this work in previous version of GitExtensions

In v2.

## Environment

- Git Extensions 3.0.2.5232
- Build 23b6f51905006ccdde8701591df706284b4155dc
- Git 2.20.1.windows.1
- Microsoft Windows NT 10.0.17134.0
- .NET Framework 4.7.3362.0
- DPI 96dpi (no scaling)

**Comment by mstv** [comment]
Could you copy the Git Extensions version and build commit from the About dialog, please?
It has a copy button:
![about](https://user-images.githubusercontent.com/36601201/55625926-6e69e000-57aa-11e9-8ae6-9acc4d36c75c.png)

**Comment by michalpaukert** [comment]
I updated issue description

**Comment by gerhardol** [comment]
> Anyway when I build DEBUG from current master and run it locally it is much faster, 

There are a number of improvements related to submodules in master, so it seems the problem is fixed

> but it opens gitextensions repository directly instead of dashboard.

Controlled by the setting below.
However, when starting in debugger, the dashboard is always skipped

![image](https://user-images.githubusercontent.com/6248932/55634119-1341e880-57be-11e9-9895-a9e64340ef46.png)

**Comment by gerhardol** [comment]
Please try latest build in master:
https://ci.appveyor.com/project/gitextensions/gitextensions/branch/master/artifacts

**Comment by AndrejSlo** [comment]
This issue should now have been closed! I've noticed the same problem since of version 3.0 and is still easily reproducible on latest 3.2 version. If you do not have internet connection you have a slow start! I've disabled update checks and the telemetry, but still this does not help!

**Comment by mstv** [comment]
I tried it with starting to the dashboard and to the last working directory, with and without avatar images.
I cannot see a difference between flight mode and internet available.
How long does it take to start GE a second and a third time, @AndrejSlo?
Which anti-virus tool do you use?

**Comment by gerhardol** [comment]
This issue was resolved, if you have a specific problem open a new issue with information like network logs.

One user reported that deactivating avatars help. Setting to author Initials should work too
![image](https://user-images.githubusercontent.com/6248932/65911310-c33e8a00-e3cc-11e9-9eaf-c4ff7980cdd8.png)

**Comment by AndrejSlo** [comment]
GE needs 20 seconds every time! I forgot to mentions, but when 3.0 came out I tried this avatar things too, but did not help and does not help not even with latest stable 3.2 release.

**Comment by RussKie** [comment]
Are you able to debug it?

**Comment by AndrejSlo** [comment]
I guess I can try some day, currently I cannot do it.

**Comment by gerhardol** [comment]
But there has been attempts to recreate the problem (do not remember who).
The Avatar was what was found. So to investigate, open a new issue with logs for the problem.
It takes more time for any maintainer to recreate than it will for you.

**Comment by RussKie** [comment]
Please post here if you get more information that could help us to debug the issue and we'll re-open it.

**Comment by AndrejSlo** [comment]
Hi. If I don't have an internet connection for more than 24 hours git extensions is waiting on a time out for some time.

**Comment by mstv** [comment]
@AndrejSlo, could you check, please, which server names are tried to resolve and/or which servers it tries to connect to? I'd use Wireshark.

**Comment by AndrejSlo** [comment]
@mstv, this is the list of all the addresses in correct order:
**ctldl.windowsupdate.com**
slscr.update.microsoft.com
ocsps.ssl.com
**crls.ssl.com**

Always immediately after I run the application it try to resolve: ctldl.windowsupdate.com
then some times this two occures: slscr.update.microsoft.com, ocsps.ssl.com
and always after third retry of crls.ssl.com application appears.

**Comment by mstv** [comment]
When I tried to open `ctldl.windowsupdate.com`, it was redirected to a page regarding MS IE.
IIRC, GE uses an embedded IE for the tab "Build Report".
This component seems to try to phone home.
You can disable the build server integration. HTH!

![grafik](https://user-images.githubusercontent.com/36601201/81985944-44cee980-9637-11ea-96f1-42effb8b34e8.png)

**Comment by AndrejSlo** [comment]
It does not help, I still see all the addresses.
